src/main/kotlin/com/clatems/clatems/users/User.kt

```diff
@@ -0,0 +1,10 @@
+package com.clatems.clatems.users
+
+import com.clatems.clatems.commons.BaseEntity
+import javax.persistence.Entity
+
+@Entity
+class User(
+  val email: String,
+  val password: String
+) : BaseEntity() {}
```

Comment:
```suggestion
@Entity
class User(
  override val id: Long? = null,
  val email: String,
  val password: String
) : BaseEntity(id) {}
```

test를 위해서 id를 전달하는 로직 추가해주세요